### PR TITLE
Update Swagger for most multiorganization & multiuser changes

### PR DESCRIPTION
**Pull request by jaredscheib**
- [x] Rebased/mergable
  - [x] Tests pass

Related to #2123 

This PR provides updates to `swagger.json` for Chronograf 1.4.0.0/1.4.3.0 concepts, including the following:
- [x] GET/POST `/chronograf/v1/organizations`
- [x] GET/DELETE/PATCH `/chronograf/v1/organizations/{id}`
- [x] GET/POST `/chronograf/v1/users`
- [x] GET/DELETE/PATCH `/chronograf/v1/users/{id}`
- [x] update or disambiguate `User` and `Users` definitions and examples with Chronograf User
- [x] update or disambiguate `Role` and `Roles` definitions and examples with Chronograf User role

### CODE REVIEW

File: server/swagger.json [hunk 1]
```diff
@@ -2256,9 +2210,652 @@
           }
         }
       }
+    },
+    "/organizations": {
+      "get": {
+        "tags": ["organizations", "multiorganization", "multiuser"],
```
Comment (goller) [inline server/swagger.json]: I'm thinking organizations and users.

File: server/swagger.json [hunk 2]
```diff
@@ -2256,9 +2210,652 @@
...
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": ["organizations", "multiorganization", "multiuser"],
+        "summary": "Create new organization",
```
Comment (goller) [inline server/swagger.json]: Probably add description.

File: server/swagger.json [hunk 3]
```diff
@@ -2256,9 +2210,652 @@
...
+            }
+          },
+          "403": {
+            "description": "Forbidden to access this route",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "422": {
+            "description": "Invalid data schema for organization",
```
Comment (goller) [inline server/swagger.json]: Perhaps add more information about differences between invalid JSON and invalid data schema.

File: server/swagger.json [hunk 4]
```diff
@@ -2256,9 +2210,652 @@
...
+          {
+            "name": "id",
+            "in": "path",
+            "type": "string",
+            "description": "ID of the organization",
+            "required": true
+          }
+        ],
+        "summary": "Retrieve a specific organization",
+        "description": "Organizations are groups of users with various role-based access-control. They are created by SuperAdmins and managed by Admins.",
```
Comment (goller) [inline server/swagger.json]: This is more the definition of organization; perhaps this should be the definition of the api.

File: server/swagger.json [hunk 5]
```diff
@@ -2256,9 +2210,652 @@
...
+      }
     }
   },
   "definitions": {
+    "Organization": {
+      "type": "object",
+      "description": "A group of users with various role-based access-control.",
+      "properties": {
+        "defaultRole": {
+          "type": "string"
```
Comment (goller) [inline server/swagger.json]: description plz

File: server/swagger.json [hunk 6]
```diff
@@ -2256,9 +2210,652 @@
...
+          }
+        }
+      }
+    },
+    "Role-Chronograf": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of role (e.g. 'member', 'viewer', 'editor', or 'admin')"
```
Comment (goller) [inline server/swagger.json]: restrict the string to an enum

File: server/swagger.json [hunk 7]
```diff
@@ -2287,8 +2884,7 @@
             "shardDuration": "7d",
             "default": true,
             "links": {
-              "self":
-                "/chronograf/v1/ousrces/1/dbs/NOAA_water_database/rps/liquid"
+              "self": "/chronograf/v1/ousrces/1/dbs/NOAA_water_database/rps/liquid"
```
Comment (goller) [inline server/swagger.json]: sources

**Comment by jaredscheib** [comment]
@goller and I discussed going ahead and merging this, and splitting the rest off into a separate issue. These are available now at #2123.

**Pull request merged**